examples/strings/strings.bal

```diff
@@ -11,78 +11,77 @@ public function main() {
     string s2 = statement.toLower();
     io:println("ToLower: " + s2);
 
-    //This compares two strings ignoring the case. It returns 'true' if the strings are equal, or 'false' if they are not equal.
+    // Compare two strings ignoring the case.
+    // Returns 'true' if the strings are equal, or 'false' if they are not equal.
     boolean isEqual =
             statement.equalsIgnoreCase("lion in town. catch the lion");
     io:println("EqualsIgnoreCase: " + isEqual);
 
-    //This returns a new string that is a substring of the specified string. You must provide the original string,
+    // Create a new string that is a substring of the specified string. You must provide the original string,
     //and the starting and ending indexes of the substring.
     string s3 = statement.substring(0, 4);
     io:println("SubString: " + s3);
 
+    // Check if the `statement` string contains the string "Lion".
     boolean contains = statement.contains("Lion");
     io:println("Contains: " + contains);
 
-    //This returns the first index of the first occurrence of the substring within the specified string.
+    // Retrieve the first index of the first occurrence of the substring "on" within the `statement` string.
```

Comment:
```suggestion
    // Retrieve the index of the first occurrence of the substring "on" within the `statement` string.
```

Reply:
First was here to say we retrieve the first (starting) index of the first occurrence IINM. Same reason for other places too.

Reply:
Then we should simply say `starting index`. IMO, the term `first index` does not really convey the idea.

Reply:
Agreed, updated.